components/accelerator/nvidia/query/query.go

```diff
@@ -461,29 +461,6 @@ func (o *Output) PrintInfo(opts ...OpOption) {
 		fmt.Printf("%s successfully checked fabric manager\n", checkMark)
 	}
 
-	if o.InfinibandClassExists && o.IbstatExists {
```

Comment:
so where do we actually call ibstats for scanning? we should remove that as well until we have https://github.com/leptonai/gpud/issues/379

Reply:
We can add this back to support https://github.com/leptonai/gpud/issues/379. This PR is getting bigger :0

Reply:
Fixed these in "gpud scan" https://github.com/leptonai/gpud/pull/380.